File: CorejavaExample/Day02Examples/src/com/sonata/Employee.java
package com.sonata;

public class Employee
{
	int empId;
	String empName;
	double empSal;
	
	Employee(){}
	
	Employee (int id , String name, double sal)
	{
		this.empId= id;
		this.empName= name;
		this.empSal=sal;
	}
	public double salCal(double HRA, double DA)
	{
		double total=this.empSal+HRA+DA;
		return total;
		
	}
	public void display()
	{
		System.out.println(empId);
		System.out.println(empName);
		System.out.println(empSal);
	}
	

}

File: Typescript_2021/Employee.ts
export class Employee {
    empId:number= 21259;
    empName:string="vivek";
    empSal:number = 24777
    constructor (i:number, n:string, s:number){
        this.empId = i;
        this.empName= n;
        this.empSal = s;
    }
    display(){
        console.log(this.empId);
        console.log(this.empName);
        console.log(this.empSal);
    }
}

var e1 = new Employee(21250, "ananth",12444);
e1.display();